docs/03-declarative-layout.md

```diff
@@ -29,7 +28,116 @@ configurable rules.
 
 @DOC_layout_WIDGET_LIST@
 
-## Placing widgets
+### The different type of widget boxes (Wibox)
+
+Awesome API the word "wibox" to describe an area of the screen filled with
```

Comment:
This sentence... reads weirdly, dunno. Should it be `The Awesome API`, `In the` or something like that? `uses the word`?

How about just mentioning that `wibox` means `widget box`, so a box that contains widgets?